test/Microsoft.TestPlatform.CommunicationUtilities.PlatformTests/SocketCommunicationManagerTests.cs

```diff
@@ -45,12 +45,14 @@ public void Dispose()
 #if NETFRAMEWORK
             // tcpClient.Close() calls tcpClient.Dispose().
             this.tcpClient?.Close();
+            GC.SuppressFinalize(this);
```

Comment:
```suggestion
```

Reply:
resolved added GC.SuppressFinalize(this); after the #endif